# Counts Tables

Sequences were processed through a qiime2 pipeline.

View the file **taxonomy-barplot.qzv** through [https://view.qiime2.org/](https://view.qiime2.org/), and see the Provenance tab for a record of all inputs and processing steps that created this file.

The **filtered-counts_level-[1-7].csv** files are derived from the taxonomy-barplot.qzv artifact.  (File names were modified.)

The **citations.bib** file is derived from the taxonomy-barplot.qzv artifact.  The file name was modified and the "|" character replaced with "-" so the citations could be rendered by citations-pretty.Rmd to create a human readable citations-pretty.html, which is pasted below.

## Citations

Bokulich, Nicholas A., Benjamin D. Kaehler, Jai Ram Rideout, Matthew Dillon, Evan Bolyen, Rob Knight, Gavin A. Huttley, and J. Gregory Caporaso. 2018. “Optimizing Taxonomic Classification of Marker-Gene Amplicon Sequences with QIIME 2’s Q2-Feature-Classifier Plugin.” Microbiome 6 (1): 90. https://doi.org/10.1186/s40168-018-0470-z.

Bolyen, Evan, Jai Ram Rideout, Matthew R. Dillon, Nicholas A. Bokulich, Christian C. Abnet, Gabriel A. Al-Ghalith, Harriet Alexander, et al. 2019b. “Reproducible, Interactive, Scalable and Extensible Microbiome Data Science Using QIIME 2.” Nature Biotechnology 37 (8): 852–57. https://doi.org/10.1038/s41587-019-0209-9.

———, et al. 2019c. “Reproducible, Interactive, Scalable and Extensible Microbiome Data Science Using QIIME 2.” Nature Biotechnology 37 (8): 852–57. https://doi.org/10.1038/s41587-019-0209-9.

———, et al. 2019a. “Reproducible, Interactive, Scalable and Extensible Microbiome Data Science Using QIIME 2.” Nature Biotechnology 37 (8): 852–57. https://doi.org/10.1038/s41587-019-0209-9.

Callahan, Benjamin J, Paul J McMurdie, Michael J Rosen, Andrew W Han, Amy Jo A Johnson, and Susan P Holmes. 2016. “DADA2: High-Resolution Sample Inference from Illumina Amplicon Data.” Nature Methods 13 (7): 581. https://doi.org/10.1038/nmeth.3869.

Hamday, Micah, and Rob Knight. 2009. “Microbial Community Profiling for Human Microbiome Projects: Tools, Techniques, and Challenges.” Genome Research 19: 1141–52. https://doi.org/10.1101/gr.085464.108.

Hamday, Micah, Jeffrey Walker J., J. Kirk Harris, Nicholas Gold J., and Rob Knight. 2008. “Error-Correcting Barcoded Primers Allow Hundreds of Samples to Be Pyrosequenced in Multiplex.” Nature Methods 5: 235--237. https://doi.org/10.1038/nmeth.1184.

McDonald, Daniel, Jose C Clemente, Justin Kuczynski, Jai Ram Rideout, Jesse Stombaugh, Doug Wendel, Andreas Wilke, et al. 2012a. “The Biological Observation Matrix (BIOM) Format or: How i Learned to Stop Worrying and Love the Ome-Ome.” GigaScience 1 (1): 7. https://doi.org/10.1186/2047-217X-1-7.

———, et al. 2012b. “The Biological Observation Matrix (BIOM) Format or: How i Learned to Stop Worrying and Love the Ome-Ome.” GigaScience 1 (1): 7. https://doi.org/10.1186/2047-217X-1-7.

McKinney, Wes. 2010c. “Data Structures for Statistical Computing in Python.” In Proceedings of the 9th Python in Science Conference, edited by Stéfan van der Walt and Jarrod Millman, 51–56.

———. 2010d. “Data Structures for Statistical Computing in Python.” In Proceedings of the 9th Python in Science Conference, edited by Stéfan van der Walt and Jarrod Millman, 51–56.

———. 2010a. “Data Structures for Statistical Computing in Python.” In Proceedings of the 9th Python in Science Conference, edited by Stéfan van der Walt and Jarrod Millman, 51–56.

———. 2010b. “Data Structures for Statistical Computing in Python.” In Proceedings of the 9th Python in Science Conference, edited by Stéfan van der Walt and Jarrod Millman, 51–56.

Pruesse, Elmar, Christian Quast, Katrin Knittel, Bernhard M Fuchs, Wolfgang Ludwig, Jorg Peplies, and Frank Oliver Glockner. 2007a. “SILVA: A Comprehensive Online Resource for Quality Checked and Aligned Ribosomal RNA Sequence Data Compatible with ARB.” Nucleic Acids Res 35 (21): 7188–96.

———. 2007b. “SILVA: A Comprehensive Online Resource for Quality Checked and Aligned Ribosomal RNA Sequence Data Compatible with ARB.” Nucleic Acids Res 35 (21): 7188–96.

Quast, Christian, Elmar Pruesse, Pelin Yilmaz, Jan Gerken, Timmy Schweer, Pablo Yarza, Jorg Peplies, and Frank Oliver Glockner. 2013a. “The SILVA Ribosomal RNA Gene Database Project: Improved Data Processing and Web-Based Tools.” Nucleic Acids Res 41 (Database issue): D590–6.

———. 2013b. “The SILVA Ribosomal RNA Gene Database Project: Improved Data Processing and Web-Based Tools.” Nucleic Acids Res 41 (Database issue): D590–6.

Rognes, Torbjørn, Tomáš Flouri, Ben Nichols, Christopher Quince, and Frédéric Mahé. 2016b. “VSEARCH: A Versatile Open Source Tool for Metagenomics.” PeerJ 4: e2584. https://doi.org/10.7717/peerj.2584.

———. 2016a. “VSEARCH: A Versatile Open Source Tool for Metagenomics.” PeerJ 4: e2584. https://doi.org/10.7717/peerj.2584.
